yarn.lock

```diff
@@ -6414,6 +6402,27 @@ amplify-codegen@^3.1.0, amplify-codegen@^3.3.1:
     semver "^7.3.5"
     slash "^3.0.0"
 
+amplify-codegen@^3.3.3:
```

Comment:
I see two versions of codegen now. Is it possible to consolidate these?

Reply:
done